source/common/router/header_formatter.h

```diff
@@ -35,18 +41,20 @@ using HeaderFormatterPtr = std::unique_ptr<HeaderFormatter>;
  */
 class StreamInfoHeaderFormatter : public HeaderFormatter {
 public:
-  StreamInfoHeaderFormatter(absl::string_view field_name, bool append);
+  StreamInfoHeaderFormatter(absl::string_view field_name, bool append, bool overwrite);
```

Comment:
Regardless the protobuf API is, we should use the enum here. 2 bools for 3 options are confusing

Reply:
Changed it to `enum` as per the suggestion. Thanks!